### PR TITLE
Development

### PR DESCRIPTION
**Pull request by moudio**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+                    <p><a href="#">Watch TNW2019 talks form Nike, Google, and more <i class="fas fa-long-arrow-alt-right"></i></a></p>
+                </div>
+</nav>
+
+
+    </header> <!--End of header-->
+    <main>
+            <section class="top-articles">
+
+                    <div class="article1">
```
Comment (Torres-ssf) [inline index.html:80]: Should be an `article` tag instead.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,738 @@
...
+                                    <li>by Napier Lopez</li>
+                                    <li><i class="far fa-clock"></i> 26 minutes ago</li>
+                                    <li><i class="fas fa-share-alt"></i> 5</li>
+                                </ul>
+                        </div>
+                       
+                    </div>
+                    <div class="second-third-articles">
+                        
+                            <div class="article2">
```
Comment (Torres-ssf) [inline index.html:101]: Should be an `article` tag instead.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,738 @@
...
+                                    
+                                    <ul class="details">
+                                        <li> by Rachel Kaser</li>
+                                        <li> <i class="far fa-clock"></i> 1 hour ago</li>
+                                        <li><i class="fas fa-share-alt"></i> 0</li>
+                                    </ul>
+                                </div>
+                            </div>
+                        
+                            <div class="article3">
```
Comment (Torres-ssf) [inline index.html]: Should be an `article` tag instead.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,738 @@
...
+                    </div>
+                
+                   
+                </section>
+            
+        <!-- LATEST NEWS -->
+
+        <h3 class="latest-news-title"><a href="#">Latest news <i class="fas fa-greater-than"></i></a></h3>
+
+        <div class="latest-news">
```
Comment (Torres-ssf) [inline index.html]: Should be a `section` tag instead.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,738 @@
...
+                </div>
+            </div>
+            
+
+        </div>
+
+
+        <!--LATEST FUNDING-->
+
+        <div class="latest-funding">
```
Comment (Torres-ssf) [inline index.html:238]: Should be a `section` tag instead.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,738 @@
...
+                </section>
+            
+        <!-- LATEST NEWS -->
+
+        <h3 class="latest-news-title"><a href="#">Latest news <i class="fas fa-greater-than"></i></a></h3>
+
+        <div class="latest-news">
+
+
+                    <div class="latest-news-thumbnail">
```
Comment (Torres-ssf) [inline index.html:146]: This should be an `article` tag instead. Each one of the 8 latest news articles must be inside an `article` tag. Kindly do that with them all.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,738 @@
...
+                <li><a href="#">Add info | </a></li>
+                <li><a href="#">Get updates by email | </a></li>
+                <li><a href="#">See more</a></li>
+            </ul>
+            <span><a href="#" class="span1 info">Add info </a> <a href="#" class="span1 info">Get updates by email </a> <a href="#" class="span1">See more</a></span>
+        </div>
+
+
+        <div class="latest-funding-details">
+                <div class="subjectwell">
```
Comment (Torres-ssf) [inline index.html]: This should be an article tag instead. Each one of the 3 latest funding articles must be inside an article tag. Kindly do that with them all.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,738 @@
...
+                         <li><a href="#"><i class="fas fa-briefcase"></i>Infinity Venture Capital</a></li>
+                        </ul>
+                        <p><a href="#">Ecommerce shipping startup banks $2.2m in pre-series B round <span class="index">on Tech in Asia</span></a></p>
+                        </div>
+                
+                </div>
+
+
+
+        <div class="group1">
```
Comment (Torres-ssf) [inline index.html]: This should be a `section` tag instead.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,738 @@
...
+            <div class="apps group-item">
```
Comment (Torres-ssf) [inline index.html]: This should be a `section` tag instead. Each one of the 8 sections (apps, gear, tech, creative, podium...) must be inside a section tag. Kindly do that with them all.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,738 @@
...
+            <h3 class="category-title"><a href="#">Apps <i class="fas fa-greater-than"></i></a></h3>
+              <div class="apps-image">
```
Comment (Torres-ssf) [inline index.html:333]: This should be an article tag instead. Each one of the 8 within articles must be inside an article tag (even the elements inside the `li` tag. Kindly do that with them all.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,738 @@
...
+                  <div class="overlay3"></div>
+                  <img src="images/apps.png" alt="mind mapping">
+                  <div class="group-item-text">
+                  <span class="image-title">MOST POPULAR</span>
+                  <h4 class="article-title"><a href="#">Report: Google Maps hosts 'millions' of fake business listings</a></h4>
+                  <span class="article-author-detail"><a href="#">Matthew Beedham</a>7 hours ago</span>
+              </div>
+            </div>
+
+              <div class="related">
```
Comment (Torres-ssf) [inline index.html]: This should be an article tag instead.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,738 @@
...
+
+                  <div class="related-details">
+                      <h5 class="related-latest">LATEST</h5><br>
+                      <span class="related-title"><a href="#">Google's new extension will help you report dubious sites with a click</a></span>
+
+                  </div>
+                  <img src="images/latest1.png" class="related-image" alt="">
+              </div>
+               <ul class="group-list">
+                   <li><a href="#">YouTube will soon let you try out makeup with AR</a></li>
```
Comment (Torres-ssf) [inline index.html:353]: The anchor tag must be inside an `article` tag. Kindly do that with all list elements.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,738 @@
...
+
+        
+
+        <div class="latest-deals">
+
+            <div class="latest-deals-thumbnail">
+                <div class="latest-deals-img">
+                <img src="images/deals-epicpxls.png" alt="Epic deals">
+            </div>
+            <div class="latest-deals-text">
```
Comment (Torres-ssf) [inline index.html:623]: Should be inside a `section` instead.

File: index.html [hunk 14]
```diff
@@ -0,0 +1,738 @@
...
+                <h3 class="deals-title"><a href="#">EpicPxls Design Assets Premium Plan: Lifetime subscription</a></h3>
+                <span class="deals-price"><span class="strike-price">$2400</span>  $99</span>
+            </div>
+            </div>
+
+            <div class="latest-deals-thumbnail">
```
Comment (Torres-ssf) [inline index.html]: Each one of the 8 articles must be inside an `article` tag instead. kindly do that with all of them.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by Torres-ssf** [comment]
Hi there @MouhaDiouf @emmanuelkamala. You guys did a really good work on this project 👏. But before I give 👍, we need to do some change to make it match Microverse guidelines. I did a series of reviews in your code. After making these changes, it will be in accord with Microverse's guidelines.
Most of these changes are needed because its best practice to use the available new tags from HTML5. They were made to make your HTML more semantic, and this is important for many reasons. Check out this article to learn more about it: https://www.lifewire.com/why-use-semantic-html-3468271

**Comment by bellom** [comment]
Hi @MouhaDiouf @emmanuelkamala,
can you please fix the validation `errors` and ignore `warnings`

**Review by bellom — APPROVED** [review]
Hi @MouhaDiouf @emmanuelkamala,
You guys have done great, i trust you guys are going to fix the validation errors.
Go ahead and merge after fixing the errors
Goodluck in the next project :+1:

**Comment by moudio** [comment]
Thank you!

Le mar. 25 juin 2019 à 23:00, Oluwaseun Bello <notifications@github.com> a
écrit :

**Pull request merged**